### ISSUE TITLE
When using InstantPreview  GvrVRHelpers.GetHeadRotation();  doesnt work anymore.

### ISSUE THREAD

**Issue by whizzkid** [googlevr/gvr-unity-sdk#865]
Summary:
When InstantPreview is working (i.e. connected to your phone) then GvrVRHelpers.GetHeadRotation(); doesnt return the correct angle anymore.

Found using:
 * Google VR SDK version: 1.120.0
 * Unity version: 2017.3.1p3
 * Phone manufacturer, model, and O/S version: Samsung galaxy s6
 * Viewer manufacturer & model:

Steps to reproduce the issue:
 1. turn on Instant Preview
 1. run your scene,  and have it connect to a phone
 1. try to access GvrVRHelpers.GetHeadRotation();  

Workarounds:
use Camera.main.transform.rotation; instead.

Additional comments:

**Comment by ddrocco** [comment]
Thanks for reporting; we'll take a look!

**Comment by whizzkid** [comment]
Also, when you use the GyroController  [https://developers.google.com/vr/develop/unity/guides/magic-window](url)  GetHeadRotation doesn't work properly because the GyroController just changes the camera's rotation.. so you'll manually have to workaround that again.

It would be cool if you could actually **Set**HeadRotation  for situations like this.

**Comment by ddrocco** [comment]
Hi, we've addressed the first problem, and it should be live with our next release on 5/28.  If you want to make a local change on your end before that, it was an issue with branching in the GetHeadRotation's editor code.  We've added the following lines inside the #if UNITY_EDITOR statement:

if (InstantPreview.Instance.IsCurrentlyConnected != null) {
  return Camera.main.transform.localRotation;
} else {
  //prior behavior
}

Regarding the second problem, we'll take a look at detecting the GyroController behavior and getting rotation accordingly.

Accommodating a SetHeadRotation method would be useful, but would require a larger refactor to work well.  We'll look into it and let you know what we decide!
